tests/components/mikrotik/__init__.py

```diff
@@ -62,48 +66,32 @@
     ".id": "*264",
     "interface": "wlan1",
     "mac-address": "00:00:00:00:00:01",
-    "ap": False,
-    "wds": False,
-    "bridge": False,
```

Comment:
What are these changes about, they don't seem to have anything to do with type hints?

Reply:
I just removed some keys from the sample response data that are not used by the integration. Just thought about doing some cleanup while adding type hints.